gatsby-config.js

```diff
@@ -42,33 +42,24 @@ module.exports = {
       resolve: `gatsby-plugin-styled-components`,
     },
     {
-      resolve: `gatsby-plugin-google-analytics`,
+      resolve: `gatsby-plugin-facebook-pixel`,
       options: {
-        trackingId: "UA-133655199-1",
-        // Puts tracking script in the head instead of the body
-        head: false,
-        // Setting this parameter is optional
-        // anonymize: true,
-        // Setting this parameter is also optional
-        // respectDNT: true,
-        // Avoids sending pageview hits from custom paths
-        // exclude: ["/preview/**", "/do-not-track/me/too/"],
-        // Enables Google Optimize using your container Id
-        // optimizeId: "YOUR_GOOGLE_OPTIMIZE_TRACKING_ID",
-        // Enables Google Optimize Experiment ID
-        // experimentId: "YOUR_GOOGLE_EXPERIMENT_ID",
-        // Set Variation ID. 0 for original 1,2,3....
-        // variationId: "YOUR_GOOGLE_OPTIMIZE_VARIATION_ID",
-        // Any additional create only fields (optional)
-        // sampleRate: 5,
-        // siteSpeedSampleRate: 10,
-        cookieDomain: "sancrisoft.com",
+        pixelId: '2607943319424233',
       },
     },
     {
-      resolve: `gatsby-plugin-facebook-pixel`,
+      resolve: "gatsby-plugin-google-tagmanager",
       options: {
-        pixelId: '2607943319424233',
+        id: "GTM-THWPGQ7",
+  
+        // Include GTM in development.
+        // Defaults to false meaning GTM will only be loaded in production.
+        includeInDevelopment: false,
+  
+        // datalayer to be set before GTM is loaded
+        // should be an object or a function that is executed in the browser
+        // Defaults to null
+        defaultDataLayer: { platform: "gatsby" },
```

Comment:
No estoy seguro que aqui diga "gatsby" o deberia decir más bien "sancrisoft site" o algo así, investigue que hace el google tagmanager con este valor para así saber mejor que nombre debemos poner

Reply:
El data Layer contiene info generica de la app

"The data layer is a map holding generic information about the application."